src/Application.vala

```diff
@@ -175,6 +177,51 @@ public class Music.Application : Gtk.Application {
         playback_manager.shuffle ();
     }
 
+    private void action_open () {
+        var all_files_filter = new Gtk.FileFilter () {
+            name = _("All files"),
+        };
+        all_files_filter.add_pattern ("*");
+        var music_files_filter = new Gtk.FileFilter () {
+            name = _("Music Files"),
```

Comment:
`All files` (Sentence case) v.s. `Music Files` (Title Case)―maybe we should unify to either one.